macros/src/test/scala/org/zalando/grafter/macros/ReaderMacroTest.scala

```diff
@@ -30,6 +30,7 @@ case class C()
 @reader[ApplicationConfig]
 case class R1(r2: R2, r3: R3, r4: R4) {
   private val field1 = "should not be used for the reader instance"
+  println(field1)
```

Comment:
we could have a comment explaining that, it will be tempting to remove it after we forget why it is there :) (although I guess the build will break ?)